### PR TITLE
Collab with Lia

### LINKED ISSUE #106: Collab with Lia / 14.09.2023
- [x] New character definition
- - [x] Base definition
- - [x] Tally image
- - [x] Sounds
- - [x] Prompts
- [x] Stream definition (incl. routing)
- [x] Index menu entry
- [x] New / updated prompts
- [x] New / updated sounds
- [x] New free block image

### PR DESCRIPTION
**Pull request by cjmaxik**
Closes #106

### CODE REVIEW

File: src/prompts/characters/lia.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+// should NOT contain any generic collab prompts
+export const regularPrompts = [
+  'talks about feet',
+  'Freudian slips',
+  'stutters',
+  'screams',
+  'speechless',
+  'lewd joke',
```
Comment (cjmaxik) [inline src/prompts/characters/lia.js:24]: We have `sussy joke` in general collab prompts

File: src/prompts/characters/lia.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  'AD time',
```
Comment (cjmaxik) [inline src/prompts/characters/lia.js:25]: She's corpa for their coffee, maybe use that?

File: src/prompts/characters/lia.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  'gets donowalled',
```
Comment (cjmaxik) [inline src/prompts/characters/lia.js:26]: We have that in Neuro's collab prompts

**Comment by cjmaxik** [comment]
@tioopental final changes are in, please review

**Review by tioopental — APPROVED** [review]

**Pull request merged**